ios/browser/skus/skus_sdk_service_factory.cc

```diff
@@ -0,0 +1,60 @@
+/* Copyright (c) 2021 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "brave/ios/browser/skus/skus_sdk_service_factory.h"
+
+#include "brave/components/skus/browser/skus_sdk_service.h"
+#include "components/keyed_service/core/keyed_service.h"
+#include "components/keyed_service/ios/browser_state_dependency_manager.h"
+#include "ios/chrome/browser/browser_state/browser_state_otr_helper.h"
+#include "ios/chrome/browser/browser_state/chrome_browser_state.h"
+#include "ios/web/public/browser_state.h"
+#include "services/network/public/cpp/shared_url_loader_factory.h"
+
+namespace skus {
+
+// static
+mojom::SkusSdk* SkusSdkServiceFactory::GetForBrowserState(
+    ChromeBrowserState* browser_state) {
+  return nullptr; // Currently don't have the correct type to return
+  // return static_cast<SkusSdkService*>(
+      // GetInstance()->GetServiceForBrowserState(browser_state, true));
```

Comment:
iOS still doesn't have a functional factory as `SkusSdkService` doesn't conform to `mojom::SkusSdk`

Reply:
this review got borked as part of a rebase - please comment in #10847

Reply:
Sorry, didn't even realize this was a PR pointing at Clifton's branch!